api/v1alpha1/openstackserver_types.go

```diff
@@ -104,6 +104,12 @@ type OpenStackServerSpec struct {
 	// be injected into the server instance.
 	// +optional
 	UserDataRef *corev1.LocalObjectReference `json:"userDataRef,omitempty"`
+
+	// SchedulerHintAdditionalProperties are arbitrary scheduler hint key/values.
```

Comment:
Would be useful to describe to end users what they might be able to achieve with this field so that when they `kubectl explain` it tells them what they can do